packages/client/src/features/modal/components/SendBookingsLinkDialog/utils.ts

```diff
@@ -78,12 +73,22 @@ interface SendBookingsLink {
     payload: {
       method: SendBookingLinkMethod;
       bookingsLink: string;
+      displayName: string;
     } & Customer
   ): FirestoreThunk;
 }
 
 export const sendBookingsLink: SendBookingsLink =
-  ({ name, method, email, surname, phone, secretKey, bookingsLink }) =>
+  ({
+    name,
+    method,
+    email,
+    surname,
+    phone,
+    secretKey,
+    bookingsLink,
+    displayName,
+  }) =>
   async (dispatch) => {
```

Comment:
This function returns a thunk, which has the following signature:
```typescript
interface FirestoreThunk {
  (
    dispatch: Dispatch // Redux dispatch wrapper
    getState: () => LocalState // Get the current state in the store
  ): Promise<void>
```
So, instead of passing a whole bunch of parameters, let's minimise them to the ones needed per function call, and have the function read from local store like so:
`const { displayName = getOrganization() } = getAboutOrganization(getState)[getOrganization()] || {}`

Also, let's avoid `__organziation__` as that's a default organization fallback and `getOrganization()` is more test-friendly way of using this (for cypress tests, where the organization is set per test basis). `getOrganization()` will still fall back to `__organization__` if no other organization is found